editor/header/preview-button/index.js

```diff
@@ -76,8 +76,8 @@ export class PreviewButton extends Component {
 
 		const markup = `
 			<div>
-				<p>Please wait&hellip;</p>
-				<p>Generating preview.</p>
+				<p>${ __( 'Please wait&hellip;' ) }</p>
```

Comment:
Unlike with React, we don't get string escaping for free here, so there is a potential risk with inserting this into raw markup.

https://codex.wordpress.org/I18n_for_WordPress_Developers#HTML

We may need to consider patterns for sanitization.

Reply:
Related: https://github.com/WordPress/packages/pull/12

Reply:
We could also generate this with `wp.element.renderToStaticMarkup` as an element.

Or just [Lodash `_.escape`](https://lodash.com/docs/4.17.5#escape)